man/i3-msg.man

```
@@ -91,6 +101,9 @@ i3-msg border normal
 
 # Dump the layout tree
 i3-msg -t get_tree
+
+# Monitor focused windows
+i3-msg -t subscribe -m '[ "window", "focus" ]'
```

Just realized that this is not correct, you can't subscribe to specific event types. What you are doing here is subscribing to the `window` event (which exists) and the `focus` event (which doesn't, but i3 silently accepts it).

We could support subscribing to specific types in i3, for example `[ "window::focus" ]`, but this is outside the scope of this PR and should be discussed in an issue first.

Also correct the commit message please.

---

Good catch, I've rewrote the commit message and the documentation. Thanks!